downstream/modules/platform/proc-editing-inventory-file.adoc

```diff
@@ -8,38 +8,55 @@
 [role="_abstract"]
 You can use the {PlatformName} installer inventory file to specify your installation scenario.
 
-
 .Procedure
 
-. Navigate to the installer:
-.. [RPM installed package]
-+
+== For an RPM installation
```

Comment:
@ianf77 
I don't think we're allowed to have subtitles inside a module for AEM.
I'm happy to merge the content if you want to address this at a different time.